src/vaadin-message-input.js

```diff
@@ -12,6 +12,9 @@ import '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
  * `<vaadin-message-input>` is a Web Component for sending messages.
  * It consists of a text area that grows on along with the content, and a send button to send message.
  *
+ * The input field can be submitted by pressing enter.
```

Comment:
Let's emphasise the supported ways of submitting the message e.g. using a list:

```
The message can be sent by one of the following actions:

- by pressing Enter (use Shift + Enter to add a new line)
- by clicking `submit` button.
```

Reply:
Done.